extensions/cornerstone/src/init.js

```diff
@@ -0,0 +1,83 @@
+import OHIF from '@ohif/core';
+import cornerstone from 'cornerstone-core';
+import csTools from 'cornerstone-tools';
+import initCornerstoneTools from './initCornerstoneTools.js';
+
+/**
+ *
+ *
+ * @export
+ * @param {*} configuration
+ */
+export default function init(configuration) {
+  const { StackManager } = OHIF.utils;
+  const metadataProvider = new OHIF.cornerstone.MetadataProvider();
+
+  cornerstone.metaData.addProvider(
+    metadataProvider.provider.bind(metadataProvider)
+  );
+
+  StackManager.setMetadataProvider(metadataProvider);
+
+  //
+  initCornerstoneTools({
```

Comment:
You should use the `configuration` here. If `configuration` is just an object, merge it, if its an array merge/add it to the correct array element: https://github.com/cornerstonejs/cornerstoneTools/blob/3f5e46c5e243ab7f27759af0df184946df283c2c/src/init.js#L25